Code/GraphMol/DistGeomHelpers/Wrap/testDistGeom.py

```diff
@@ -367,6 +367,12 @@ def test7ConstrainedEmbedding(self):
 
 
     def test8MultiThreadMultiConf(self):
+        if (rdBase.rdkitBuild.split('|')[2] != "MINGW"):
```

Comment:
I think this will work.  I may explore to see if this is a 32 vs 64 bit issue.  This doesn't seem likely if we are using double throughout, but you never know.